src/rprblender/nodes/blender_nodes.py

```diff
@@ -969,6 +969,77 @@ def _create_transmission_node(self, base_color, roughness):
         return rpr_node
 
 
+class ShaderNodeBsdfHairPrincipled(NodeParser):
+    """ Partial support for Cycles Principled Hair BSDF shader node """
+
+    def export(self) -> [NodeItem, None]:
+        parametrization = self.node.parametrization
+
+        roughness = self.get_input_value('Roughness').clamp(0.001, 1.0)
+        roughness_radial = self.get_input_value('Radial Roughness').clamp(0.001, 1.0)
+        coat = self.get_input_scalar('Coat')
+        ior = self.get_input_scalar('IOR')
+
+        if parametrization == 'ABSORPTION':
+            absorption_color = self.get_input_scalar('Absorption Coefficient')
+            color = (1.0 - absorption_color).min(1.0)
+        elif parametrization == 'MELANIN':
+            melanin = self.get_input_scalar('Melanin')
+            melanin_redness = self.get_input_scalar('Melanin Redness')
+            absorption_color = self._calculate_absorption_from_melanin(melanin, melanin_redness)
+            color = (1.0 - absorption_color).min(1.0)
+        else:  # 'COLOR'
+            color = self.get_input_scalar('Color')
+            absorption_color = self._calculate_absorption_from_color(color, roughness_radial)
+
+        node = self._create_absorption_node(color, absorption_color, roughness, roughness_radial, coat, ior)
+        return node
+
+    def _calculate_absorption_from_color(self, color, roughness_radial):
+        """ Use the Cycles way from bsdf_principled_hair_albedo_roughness_scale and bsdf_principled_hair_sigma_from_reflectance """
+        x = roughness_radial
+        roughness_scale = (((((0.245 * x) + 5.574) * x - 10.73) * x + 2.532) * x - 0.215) * x + 5.969
```

Comment:
Where did this calculation come from

Reply:
Cycles code and Blender Manual https://docs.blender.org/manual/en/latest/render/shader_nodes/shader/hair_principled.html